### ISSUE TITLE
Add TriplyDB

### ISSUE THREAD

**Issue by regisrob** [IIIF/awesome-iiif#419]
https://triply.cc/blog/2023-06-iiif/

But no idea where to put it... (it is not a CMS, nor a kind of CMSs or DAMs).

**Comment by hadro** [comment]
I agree with adding to the list, though also agree about the difficulty of category -- perhaps a "FAIR data provider" or just a "data provider" category might make sense, or something similar, if we could find another item to fit under the same heading?

**Comment by kirschbombe** [comment]
I would probably call it a DMS (data management system). Maybe we can rename the CMS/DAMS section to include DMS or something? In any case, I've not seen this before and plan to check it out, so thanks for bringing it to the list @regisrob :) Or a separate DMS section that might also include things like Recogito.
